components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+/** States used to configure bottom navigation on when to hide item titles.  */
+typedef NS_ENUM(NSInteger, MDCBottomNavigationBarTitleHideState) {
+
+  // Default behavior is to hide item titles when item is unselected.
+  MDCBottomNavigationBarTitleHideStateDefault = 0,
+
+  // Item titles are never hidden.
+  MDCBottomNavigationBarTitleHideStateNever = 1,
+
+  // Item titles are always hidden.
+  MDCBottomNavigationBarTitleHideStateAlways = 2
+};
+
+/**
+ A bottom navigation bar.
+
+ The bottom navigation bar is docked at the bottom of the screen with tappable items. Only one item
+ can be selected at at time. The selected item's title text is displayed. Title text for unselected
+ items are hidden.
+ */
+@interface MDCBottomNavigationBar : UIView
+
+/**
+ Configures when item titles should be displayed.
+ Default is MDCBottomNavigationBarTitleHideStateUnselect.
```

Comment:
Consider:
Default behavior is to only show the title of the selected item.
(MDCBottomNavigationBarTitleHideStateUnselect doesn't exist)